test/e2e/framework/networking/config.go

```diff
@@ -0,0 +1,688 @@
+package networking
```

Comment:
fixtures.go? :P 

Also missing licence header

Reply:
Yeah, I'll fix this, please take a look at the comment above: 🤣 

> I start to work on the refactor of networking utils and found out several problems.
>
> + networking pkg still needs Failf function not implemented in the framework package which will lead to import cycle problem.
> + the shuffleNodes function was both used in networking and nodes package.
> + I don't have a good idea on how to deal with the functions named with Test prefix.
>
> Do you have thoughts on this? Thank you.

Reply:
> networking pkg still needs Failf function not implemented in the framework package which will lead to import cycle problem.

https://github.com/kubernetes/kubernetes/pull/77160 is currently refactoring wrapper functions like `Failf` but it doesn't move it to it's own package, just a dedicated file under `test/e2e/framework`. That will still cause import cycles here right?

Reply:
Yeah, we need to move it to a sub package in framework to avoid it.

Reply:
Looks like the only reason we import `e2enet` in `test/e2e/framework/networking_utils.go` is for `PokeHTTP`, if we removed that import somehow would that resolve the import cycle for now?

Reply:
Yeah, do you have any suggestions on where to place these functions?

Reply:
`test/e2e/framework/http`?

Reply:
Ah, that won't work cause PokeHTTP will depend on framework. Can we just leave PokeHTTP in networking_utils.go for now?